lib/v2/screens/transfer/send_confirmation/interactor/viewmodels/send_confirmation_state.dart

```diff
@@ -1,23 +1,24 @@
 import 'package:equatable/equatable.dart';
 import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/screens/transfer/send_confirmation/interactor/viewmodels/send_confirmation_arguments.dart';
 import 'package:seeds/v2/screens/transfer/send_confirmation/interactor/viewmodels/send_confirmation_commands.dart';
 import 'package:seeds/v2/screens/transfer/send_confirmation/interactor/viewmodels/send_info_line_items.dart';
 
 class SendConfirmationState extends Equatable {
   final PageState pageState;
-  final String? account;
-  final String? name;
-  final Map<String, dynamic>? data;
-  final List<SendInfoLineItems>? lineItems;
+  final String account;
+  final String name;
+  final Map<String, dynamic> data;
+  final List<SendInfoLineItems> lineItems;
   final String? error;
   final ShowTransactionSuccess? pageCommand;
 
   const SendConfirmationState({
```

Comment:
i know this page cant load unless we have those arguments, so make them none nullable